python/mxnet/ndarray/numpy/linalg.py

```diff
@@ -19,50 +19,184 @@
 
 from __future__ import absolute_import
 from . import _op as _mx_nd_np
+from . import _internal as _npi
+
 
 __all__ = ['norm']
 
 
+# pylint: disable=line-too-long, too-many-return-statements, too-many-branches, no-member, consider-using-in, no-else-return
```

Comment:
Please don't just disable these rules.

Reply:
ok, i will modify my operator after op svd  and min  are merged